stop-mycroft.sh

```diff
@@ -87,33 +88,33 @@ result=0  # default, no change
 
 
 OPT=$1
-shift
+if [ $# -gt 0]; then
```

Comment:
little typo, missing the space between `0 ]`

Reply:
:see_no_evil: